jsonnet/kube-prometheus/node-exporter/node-exporter.libsonnet

```diff
@@ -155,14 +155,15 @@ local k = import 'ksonnet/ksonnet.beta.4/k.libsonnet';
           name: 'node-exporter',
           namespace: $._config.namespace,
           labels: {
-            'k8s-app': 'node-exporter',
+            'app.kubernetes.io/name': 'node-exporter',
+            'app.kubernetes.io/version': $._config.versions.nodeExporter,
           },
         },
         spec: {
           jobLabel: 'k8s-app',
           selector: {
             matchLabels: {
-              'k8s-app': 'node-exporter',
+              'app.kubernetes.io/name': 'node-exporter',
```

Comment:
The node exporter service doesn't have these labels I think. We need to change that as well.